docker-compose.yml

```diff
@@ -102,7 +102,7 @@ services:
 
   transmart-api-server:
     container_name: ${INSTANCE_ID:?Please configure INSTANCE_ID}-transmart-api-server
-    image: thehyve/transmart-api-server:17.2.5
+    build: ../transmart-core/docker/transmart-api-server
```

Comment:
Could you revert this change?

Reply:
I must have forgotten to take that out from testing.